app/admin.js

```diff
@@ -368,11 +375,32 @@ const handleVerification = co(function *(args) {
   }
 });
 
+const handleRecoveryInfo = co(function *(args){
+  const json = JSON.parse(fs.readFileSync(args.file));
+  const pub = json.backupKey;
+  const walletkey = yield WalletKey.findOne({ pub });
+  const masterkey = yield MasterKey.findOne({ pub: walletkey.masterKey });
+  json.masterkey = masterkey.pub;
+  json.masterkeypath = masterkey.path;
+  json.walletkeypath = walletkey.path;
+  const filename = args.file.substring(0,args.file.length - 5) + '-prepared.json';
+  console.log('got info... writing file ' + filename);
+  fs.writeFileSync(filename, JSON.stringify(json, null, 2));
+  console.log('done.');
+});
+
+const requireDB = function() {
+  MasterKey = require('./models/masterkey.js');
+  WalletKey = require('./models/walletkey.js');
+  db = require('./db.js');
+}
+
 const run = co(function *(testArgs) {
   const args = parser.parseArgs(testArgs);
 
   switch (args.cmd) {
```

Comment:
Below are all the admin commands. As you can see, requireDB is called for commands that require a database connection